datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/deser/MonthEnumDeserializerModifier.java

```diff
@@ -0,0 +1,60 @@
+package com.fasterxml.jackson.datatype.jsr310.deser;
+
+import java.io.IOException;
+import java.time.Month;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.util.JacksonFeatureSet;
+import com.fasterxml.jackson.databind.BeanDescription;
+import com.fasterxml.jackson.databind.DeserializationConfig;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.deser.BeanDeserializerModifier;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeFeature;
+
+
+public class MonthEnumDeserializerModifier extends BeanDeserializerModifier {
+    private final boolean _oneBaseMonths;
+
+    public MonthEnumDeserializerModifier() {
+        this(false);
+    }
+
+    public MonthEnumDeserializerModifier(boolean oneBaseMonths) {
+        _oneBaseMonths = oneBaseMonths;
+    }
+
+    public MonthEnumDeserializerModifier withFeatures(JacksonFeatureSet<JavaTimeFeature> features) {
```

Comment:
Let's change this to be passed in constructor: it's an internal implementation class, no need for fluent-style, can be modified without worries about compatibility.